iothub_client/src/iothub_client_ll_uploadtoblob.c

```diff
@@ -788,23 +787,38 @@ IOTHUB_CLIENT_RESULT IoTHubClient_LL_UploadMultipleBlocksToBlob_Impl(IOTHUB_CLIE
                                             /*Codes_SRS_IOTHUBCLIENT_LL_99_008: [ If step 2 is aborted by the client, then the HTTP message body shall look like:  ]*/
                                             LogInfo("Blob_UploadFromSasUri aborted file upload");
 
-                                            if (BUFFER_build(responseToIoTHub, (const unsigned char*)FILE_UPLOAD_ABORTED_BODY, sizeof(FILE_UPLOAD_ABORTED_BODY) / sizeof(FILE_UPLOAD_ABORTED_BODY[0])) == 0)
+                                            STRING_HANDLE aborted_response = STRING_construct_sprintf(RESPONSE_BODY_FORMAT,
+                                                                                        STRING_c_str(correlationId),
+                                                                                        (const unsigned char*)"false",
+                                                                                        (const unsigned char*)"-1",
+                                                                                        (const unsigned char*)"file upload aborted");
```

Comment:
Please create global constants for these strings.

Reply:
changed thanks